islandora_openseadragon.install

```diff
@@ -38,3 +54,21 @@ function islandora_openseadragon_uninstall() {
   );
   array_walk($variables, 'variable_del');
 }
+
+/**
+ * Implements hook_update_N().
```

Comment:
[Comments for update hooks are different](https://www.drupal.org/node/1354#updateN).